data/avro-schema/src/main/resources/avro/net/corda/data/membership/db/response/query/MemberInfoQueryResponse.avsc

```diff
@@ -11,6 +11,18 @@
         "type": "array",
         "items": "net.corda.data.membership.PersistentMemberInfo"
       }
+    },
+    {
+      "name": "signedMembers",
+      "doc": "Member Infos with signature representing members found for a given query.",
+      "type": [
+        "null",
+        {
+          "type": "array",
+          "items": "net.corda.data.membership.PersistentSignedMemberInfo"
```

Comment:
Can we merge the PersistenceMemberInfo and PersistenceSignedMemberInfo now that the key value pair lists are in their serialised forms?